request-management-api/request_api/resources/foidocument.py

```diff
@@ -73,6 +74,14 @@ def post(requestid, requesttype):
             requestjson = request.get_json() 
             documentschema = CreateDocumentSchema().load(requestjson)
             result = documentservice().createrequestdocument(requestid, documentschema, AuthHelper.getuserid(), requesttype)
+            # Add the RRT event here
+            print("THe version output is")
+            print(result.args[0])
+            ministryversion = result.args[0]
+            for document in documentschema['documents']:
+                # Add attachment event here as we need to pass in the document
+                # to the event service to identify if the document is an RRT document.
+                eventservice().attachmenteventservice(requestid, document, AuthHelper.getuserid(), ministryversion)
```

Comment:
-Do not pass version to the event class. Leverage the below loc for getting version in the attachment event class.
version = FOIMinistryRequest.getversionforrequest(requestid)

Reference: divisionevent.class